domain/cloud/state/state.go

```diff
@@ -293,7 +293,8 @@ ON CONFLICT(region_uuid, key) DO UPDATE
 	})
 }
 
-func loadClouds(ctx context.Context, tx *sql.Tx, name string) ([]cloud.Cloud, error) {
+// LoadClouds fetches the cloud with the given name, or all clouds if name is empty.
+func LoadClouds(ctx context.Context, tx *sql.Tx, name string) ([]cloud.Cloud, error) {
```

Comment:
because it's used in the credential state package to load the cloud for a credential and i didn't want to cut and paste the same logic elsewhere. it's a cloud state concern